railties/lib/rails/commands/server/server_command.rb

```diff
@@ -112,12 +112,18 @@ class ServerCommand < Base # :nodoc:
         desc: "Specifies the PID file."
       class_option "dev-caching", aliases: "-C", type: :boolean, default: nil,
         desc: "Specifies whether to perform caching in development."
+      class_option "log-stdout", aliases: "-l", type: :boolean, default: nil,
```

Comment:
Thor automatically dasherizes options for the CLI. So let's call this `:log_stdout`, which makes it easier to access in `options` as well.

Reply:
👍 After rebasing I saw this is used for other options as well, so amended it.